apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/MainStatSelectionCard.tsx

```diff
@@ -9,47 +8,91 @@ import { StatColoredWithUnit } from '../../../../../Components/StatDisplay';
+  const { database } = useContext(DatabaseContext)
+  const tots = useMemo(() => objectKeyMap(allSlotKeys, slotKey =>
+    catTotal(Artifact.slotMainStats(slotKey), ct => Object.entries(database.arts.data).forEach(([id, a]) => {
+      const sk = a.slotKey
+      if (sk !== slotKey) return
+      const msk = a.mainStatKey
+      if (!msk || !ct[msk]) return
+      ct[msk].total++
+      if (filteredArtIds.includes(id)) ct[msk].current++
+    }))
+  ), [filteredArtIds, database])
+  const slotTots = useMemo(() => catTotal(artifactsSlotsToSelectMainStats, ct => Object.entries(database.arts.data).forEach(([id, a]) => {
+    const sk = a.slotKey
+    if (!ct[sk]) return
+    ct[sk].total++
+    if (filteredArtIds.includes(id)) ct[sk].current++
+  })), [filteredArtIds, database])
 
-  return <Box display="flex" flexDirection="column" gap={1}>
+  return <Box display="flex" flexDirection="column">
+    <Divider />
+    <Box display="flex">
+      <CardContent sx={{ flexGrow: 1 }}>
+        <Box sx={{ display: "flex", gap: 1, alignItems: "center" }}>
+          <BootstrapTooltip placement="top" title={<Typography>{t(`slotName.flower`)}</Typography>}>
+            <span>{artifactSlotIcon("flower")}</span>
+          </BootstrapTooltip>
+          <Box flexGrow={1}>
+            <SqBadge color="info">{StatIcon.hp} {tots.flower.hp}</SqBadge>
+          </Box>
+        </Box>
+      </CardContent>
+      <Divider orientation="vertical" flexItem />
+      <CardContent sx={{ flexGrow: 1 }}>
+        <Box sx={{ display: "flex", gap: 1, alignItems: "center" }}>
+          <BootstrapTooltip placement="top" title={<Typography>{t(`slotName.plume`)}</Typography>}>
+            <span>{artifactSlotIcon("plume")}</span>
+          </BootstrapTooltip>
+          <Box flexGrow={1}>
+            <SqBadge color="info">{StatIcon.atk} {tots.plume.atk}</SqBadge>
+          </Box>
+        </Box>
+      </CardContent>
+    </Box>
     {artifactsSlotsToSelectMainStats.map(slotKey => {
-      const numSel = mainStatKeys[slotKey].length
+      const selectedMainKeys = mainStatKeys[slotKey]
+      const mainKeys = Artifact.slotMainStats(slotKey)
+      const mainKeysHandler = handleMultiSelect([...mainKeys])
       return <Box key={slotKey}>
         <Divider />
-        <CardContent sx={{ pt: 1 }}>
+        <CardContent sx={{ pt: 1, pb: 1 }}>
           <Box sx={{ display: "flex", gap: 1, alignItems: "center", pb: 1 }}>
             <BootstrapTooltip placement="top" title={<Typography>{t(`slotName.${slotKey}`)}</Typography>}>
               <span>{artifactSlotIcon(slotKey)}</span>
             </BootstrapTooltip>
             <Box flexGrow={1}>
-              <SqBadge color="info">{numSel ? `${numSel} Selected` : `Any`}</SqBadge>
+              <SqBadge color="info">{slotTots[slotKey]}</SqBadge>
             </Box>
-            <Button color="error" size="small" disabled={!mainStatKeys[slotKey].length || disabled} sx={{ mt: -1, mb: -1 }}
```

Comment:
Why remove the reset button?

Reply:
the main stat selection is now a "smart toggle", so the reset button is extra clutter. Also, reset sets everything to "off", there is no "off" in smart toggle